controllers/articles.controller.js

```diff
@@ -0,0 +1,19 @@
+const { fetchArticle } = require("../models/index")
+
+exports.getArticle = (request, response, next) => {
+    const { article_id } = request.params
+    if (isNaN(article_id)){
+        return response.status(400).send({ msg: `400: Bad Request`})
+    }
+    fetchArticle(article_id)
+    .then((article) => {
```

Comment:
If fetchArticle fails the catch block will catch the err
custom errors are probably better put in the model themselves